go/exocom/main_test.go

```diff
@@ -196,6 +215,40 @@ func FeatureContext(s *godog.Suite) {
 		}
 		return nil
 	})
+
+	s.Step(`^ExoCom broadcasts the message "([^"]*)" to the "([^"]*)" service$`, func(messageName, serviceName string) error {
+		//TODO: chekc that the received message matches the given message
+		_, err := services[serviceName].WaitForMessageWithName(messageName)
+		if err != nil {
+			return err
+		}
+		return nil
+	})
+
+	s.Step(`^the "([^"]*)" service sends "([^"]*)" in reply to "([^"]*)"$`, func(serviceName, replyMessage, responseTo string) error {
+		lastSentMessageName = replyMessage
+		return services[serviceName].Send(structs.Message{
+			Sender:     serviceName,
+			Payload:    "",
+			ID:         "123",
+			ResponseTo: responseTo,
+			Name:       replyMessage,
+		})
+	})
+
+	s.Step(`^ExoCom broadcasts the reply "([^"]*)" to the "([^"]*)" service$`, func(messageName, serviceName string) error {
+		//TODO: chekc that the received message matches the given message
+		_, err := services[serviceName].WaitForMessageWithName(messageName)
+		if err != nil {
+			return err
+		}
+		return nil
+	})
```

Comment:
This step definition is very similar to `ExoCom broadcasts the message ...`, since we aren't doing anything different I think we update the gherkin to just use one.